spring-cloud-deployer-kubernetes/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/DeploymentPropertiesResolver.java

```diff
@@ -196,11 +211,22 @@ Map<String, Quantity> deduceResourceLimits(Map<String, String> kubernetesDeploye
 		if (StringUtils.hasText(cpu)) {
 			limits.put("cpu", new Quantity(cpu));
 		}
+		if(StringUtils.hasText(ephemeralStorage)) {
```

Comment:
**[TinyNit]** formatting is inconsistent w/ existing code (spacing). I think you need to setup your formatters for SCD.

Reply:
Updated.